src/pages/Aggregator/Aggregator.component.tsx

```diff
@@ -94,6 +108,13 @@ export const AggregatorComponent = ({
     }
   }, [amount, getReceiveAmount, setValue]);
 
+  useEffect(() => {
+    resetField(AggregatorInputs.SendAmount);
+    setValue(AggregatorInputs.SendAmount, '');
```

Comment:
Do you really need to call `setValue` after `resetField`? `resetField` should reset value to initialValue and re-validate

https://react-hook-form.com/api/useform/resetfield#main